### PR TITLE
add manual doc aliases

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

**Review by GuillaumeGomez — APPROVED** [review]
Seems like my script missed quite a lot... Thanks!

**Comment by bilelmoussaoui** [comment]
More are coming!

**Comment by bilelmoussaoui** [comment]
Ok I think have done enough for now

File: glib/src/subclass/types.rs [hunk 1]
```diff
@@ -19,6 +19,7 @@ pub struct InitializingType<T>(pub(crate) Type, pub(crate) marker::PhantomData<*
 
 impl<T: ObjectSubclass> InitializingType<T> {
     /// Adds an interface implementation for `I` to the type.
+    #[doc(alias = "g_type_add_interface_static")]
```
Comment (bilelmoussaoui) [inline glib/src/subclass/types.rs:22]: @sdroege regarding this one, does it make sense to have doc aliases for subclassing support as well? :)

Comment (sdroege) [reply glib/src/subclass/types.rs:22]: Sure why not

Comment (bilelmoussaoui) [reply glib/src/subclass/types.rs:22]: I will do another round in a separate MR later then :)

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**